man/neptune_file_as_html.Rd

```diff
@@ -0,0 +1,46 @@
+\name{neptune_file_as_html}
+\alias{neptune_file_as_html}
+\title{
+  Converts an object to an HTML File value object.
+}
+\description{
+  Converts an object to an HTML File value object. This way you can upload ggplot2, Plotly, Altair, Bokeh, Matplotlib interactive charts or upload directly DataFrame objects to explore them in Neptune UI.
```

Comment:
This should mention only objects that can exist in R. So ggplot2 and Plotly? And DataFrames?

Reply:
@needlecapter this one was missed